packages/edit-site/src/components/block-editor/resizable-editor.js

```diff
@@ -84,9 +91,7 @@ function ResizableEditor( { enableResizing, settings, children, ...props } ) {
 				);
 				// Observing the <html> rather than the <body> because the latter
 				// gets destroyed and remounted after initialization in <Iframe>.
-				resizeObserver.observe(
-					iframe.contentDocument.documentElement
-				);
+				resizeObserver.observe( iframe.contentDocument.body );
```

Comment:
Should we remove the above comment since we don't observe `html` anymore?

Reply:
Thanks for spotting that, I've pushed a commit that updates the comment.

Reply:
Thanks